### ISSUE TITLE
Querying host:port tuples of ReplicaSet

### ISSUE THREAD

**Issue by problame** [KIT-MAMID/mamid#21]
As the administrator, there needs to be a comfortable way to retrieve the members of a ReplicaSet, preferrably by the ReplicaSet's `name`.

Rough draft:

```
GET /api/replicaSet/<id>/members
[
"host1:18080",
"host2:18082",
"host3:18081"
]
```

Alternatively, we could extend the already present `replicaSet` API endpoint.

The other question is whether this needs to be present in the GUI or an API endpoint (for automation tools, etc) is sufficient.

Finally: how do we manage the case where Mamid changes members of a ReplicaSet. Do we send an e-mail report, is it the admin's responsibility? This should be documented in a Manual at least....

**Comment by bwoebi** [comment]
Why do we need to report ReplicaSet member changes? These should be fully transparent I think?

**Comment by problame** [comment]
The client application / client MongoDB driver naturally needs to connect to all members of the replica set.

**Comment by problame** [comment]
Fixed as of 980b88d03d5f9ec97d6df9785a354d722c8fd623
